README.md

```diff
@@ -6,27 +6,96 @@
 
 # streams-bootstrap
 
+`streams-bootstrap` provides base classes and utility functions for Kafka Streams applications.
+
+It provides a common way to
+- configure Kafka Streams applications
+- deploy streaming applications into a Kafka cluster on Kubernetes via Helm Charts
+- reprocess data
+
+Visit our [blogpost](https://medium.com/bakdata/continuous-nlp-pipelines-with-python-java-and-apache-kafka-f6903e7e429d) and [demo](https://github.com/bakdata/common-kafka-streams-demo) for an overview and a demo application.  
+The common configuration and deployments on Kubernetes are supported by the [Streams Explorer](https://github.com/bakdata/streams-explorer), which makes it possible to explore and monitor data pipelines in Apache Kafka.
+
 ## Getting Started
 
 You can add streams-bootstrap via Maven Central.
 
 #### Gradle
+
 ```gradle
-compile group: 'com.bakdata.kafka', name: 'streams-bootstrap', version: '1.6.0'
+compile group: 'com.bakdata.kafka', name: 'streams-bootstrap', version: '1.7.0'
 ```
 
 #### Maven
+
 ```xml
 <dependency>
     <groupId>com.bakdata.kafka</groupId>
     <artifactId>streams-bootstrap</artifactId>
-    <version>1.6.0</version>
+    <version>1.7.0</version>
 </dependency>
 ```
 
-
 For other build tools or versions, refer to the [latest version in MvnRepository](https://mvnrepository.com/artifact/com.bakdata.kafka/streams-bootstrap/latest).
 
+### Usage
+
+Create a subclass of `KafkaStreamsApplication` and implement the abstract methods `buildTopology()` and `getUniqueAppId()`. You can define the topology of your application in `buildTopology()`. 
+
+```java
+import org.apache.kafka.streams.StreamsBuilder;
+import org.apache.kafka.streams.kstream.KStream;
+
+public class StreamsBootstrapApplication extends KafkaStreamsApplication {
+    public static void main(final String[] args) {
+        startApplication(new StreamsBootstrapApplication(), args);
+    }
+
+    @Override
+    public void buildTopology(final StreamsBuilder builder) {
+        final KStream<String, String> input =
+                builder.<String, String>stream(this.getInputTopics());
+        
+        // your topology
+
+        input.to(this.getOutputTopic());
+    }
+
+    @Override
+    public String getUniqueAppId() {
+        return "streams-bootstrap-app";
+    }
+
+}
+```
```

Comment:
Maybe override kafka properties but comment that it is optional?

Reply:
Like this?